symplyphysics/laws/waves/vector/phase_velocity_from_angular_velocity_and_wavevector.py

```diff
@@ -0,0 +1,74 @@
+from symplyphysics import (
+    units,
+    angle_type,
+    Symbol,
+    Quantity,
+    Vector,
+    QuantityVector,
+    validate_input,
+    validate_output,
+    scale_vector,
+    vector_magnitude,
+)
+
+# Description
+## The phase velocity of a wave is the rate at which the wave propagates in a medium.
+## It is the velocity at which the phase of one frequency component of the wave travels.
+## The phase velocity is collinear with the wavevector.
+
+# Law: v = w * k / |k|**2
+## v - vector of phase velocity of wave
+## w - angular frequency of wave
+## k - wavevector of wave
+## |k| - wavenumber of wave, where |v| is the Euclidean norm of vector v
```

Comment:
Is it the same parameter as in https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/definitions/angular_wavenumber_is_inverse_wavelength.py#L19?
I suggest to use the same name for both.

Reply:
I'll add "angular" to both of them